### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by danrovito**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,21 @@
-interactive-make
+# Interactive Make for Laravel 5.4
+
+## Getting Started
+
+To get started you must first install the package from composer.
+
+```
```
Comment (svenluijten) [inline README.md]: append `bash` here and you'll get syntax highlighting (if available) and it'll be more a11y 🙂

File: README.md [hunk 2]
```diff
@@ -1 +1,21 @@
...
+
+Add the package service provider to your `config/app.php`
+
+```php
+Laracademy\Commands\MakeServiceProvider::class,
+```
+
+Run this command in your terminal then follow the prompts
+
+```
```
Comment (svenluijten) [inline README.md]: same here

File: README.md (hunk 1 above)
Comment (danrovito) [reply README.md]: <3

File: README.md (hunk 2 above)
Comment (danrovito) [reply README.md]: <3

**Pull request merged**